sdk/cognitiveservices/ci.yml

```diff
@@ -23,16 +25,21 @@ pr:
   paths:
     include:
       - sdk/cognitiveservices/ci.yml
+      - sdk/cognitiveservices/azure-ai-speech-transcription/
```

Comment:
This should also have `parameters` node to enable batch release. See [this example](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/appconfiguration/ci.yml#L40-L48)